css-in-javascript/README.md

```diff
@@ -1,6 +1,6 @@
 # Airbnb CSS-in-JavaScript Style Guide
 
-*A mostly reasonable approach to CSS-in-JavaScript*
+_A mostly reasonable approach to CSS-in-JavaScript_
```

Comment:
Please revert unrelated formatting changes, and don’t run prettier or autoformatting on a file if the repo isn’t using that formatted already.

Reply:
Sorry for not checking above pr properly. I have closed this pr and raised another one. I have disabled the autoformatting. please check latest pr.
https://github.com/airbnb/javascript/pull/3053

Reply:
Please don’t make extra PRs - this one should have been updated.